src/gentropy/assets/schemas/variant_index.json

```diff
@@ -184,6 +214,12 @@
         "type": "array"
       }
     },
+    {
+      "metadata": {},
+      "name": "hgvsId",
+      "nullable": true,
```

Comment:
I assume it can be nullable, because we may have no annotation for some specific variant in offline mode?

Reply:
We should have hgvs identifiers for all real variants. When defining the schema, I'm quite permissive with nullability. Because when datasets are read from parquet the nullability information is lost, I tend to set the flag true for only those columns that are really fundamental, being very stringent would make testing rather complicated if there are a large number of non-nullable columns.